api/backend/ecs/utilities.go

```diff
@@ -187,10 +188,6 @@ var GetLogs = func(cloudWatchLogs cloudwatchlogs.Provider, taskARNs []*string, t
 
 	logFiles := []*models.LogFile{}
 	for _, logStream := range logStreams {
-		if *logStream.StoredBytes == 0 {
```

Comment:
how are we sure this wont re-introduce the json marshal error when the stream is empty?